Update ee13b5 and 1a02b0 rules to clarify that transcript for video content does not need to be visible (#2187)
auto-wcag/auto-wcag: _rules/video-only-transcript-ee13b5.md

    failed: not satisfied
    passed: further testing needed
    inapplicable: further testing needed
  wcag20:1.3.1: # Info and Relationships (A)

>     secondary: This success criterion is **related** to this rule. This is because the absence of a transcript means that information conveyed through visual means in the video-only content remains inaccessible to certain users, thus creating inequality in communication.

Reviewer: ```suggestion
    secondary: This success criterion is **related** to this rule. This is because this criterion applies to a visible transcript. 
```

</html>
```

#### Passed Example 3

This `video` element, which has no audio, has a non-visible text transcript available on the same page.

```html
<html lang="en">
<video controls>
  <source src="/test-assets/rabbit-video/silent.mp4" type="video/mp4"></source>
  <source src="/test-assets/rabbit-video/silent.webm" type="video/webm"></source>
</video>

> <p style="position:absolute; left:-9999px; top:auto; width:1px; height:1px; overflow:hidden;">The above video shows a giant fat rabbit climbing out of a hole in the ground.

Reviewer 2: "Above" feels a bit weird on invisible transcripts 🤔 (also on other rules).
